2024-12-11 03:29:12,947 - INFO - Finished! Total sfe time: 10.445204496383667 
2024-12-11 03:29:12,947 - DEBUG - 870,server, Parsed server ram usage: 79848
2024-12-11 03:29:12,947 - DEBUG - 870,client, Parsed client ram usage: 80860
2024-12-11 03:29:32,390 - DEBUG - client all values : ['modprobe cpufreq_stats failedLoaded 0 prior measurements', 'RAPL device for cpu 0', 'RAPL device for cpu 0', 'Devfreq not enabled', 'glob returned GLOB_ABORTED', 'Preparing to take measurements', 'Taking 1 measurement(s) for a duration of 11 second(s) each.', '2024-12-11 03:27:44,680 - tno.mpc.communication.httphandlers - INFO - Serving on 192.168.1.1:61002', '2024-12-11 03:27:44,683 - tno.mpc.communication.httphandlers - ERROR - Message not received.', 'Traceback (most recent call last):', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1061, in _wrap_create_connection', '    sock = await aiohappyeyeballs.start_connection(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 103, in start_connection', '    raise first_exception', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 81, in start_connection', '    sock = await _connect_sock(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohappyeyeballs/impl.py", line 165, in _connect_sock', '    await loop.sock_connect(sock, address)', '  File "/usr/lib/python3.9/asyncio/selector_events.py", line 502, in sock_connect', '    return await fut', '  File "/usr/lib/python3.9/asyncio/selector_events.py", line 537, in _sock_connect_cb', "    raise OSError(err, f'Connect call failed {address}')", "ConnectionRefusedError: [Errno 111] Connect call failed ('192.168.1.2', 61001)", '', 'The above exception was the direct cause of the following exception:', '', 'Traceback (most recent call last):', '  File "/home/kamil/.local/lib/python3.9/site-packages/tno/mpc/communication/httphandlers.py", line 193, in _send', '    async with self.session.post(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/client.py", line 1355, in __aenter__', '    self._resp: _RetType = await self._coro', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/client.py", line 659, in _request', '    conn = await self._connector.connect(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 557, in connect', '    proto = await self._create_connection(req, traces, timeout)', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1002, in _create_connection', '    _, proto = await self._create_direct_connection(req, traces, timeout)', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1336, in _create_direct_connection', '    raise last_exc', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1305, in _create_direct_connection', '    transp, proto = await self._wrap_create_connection(', '  File "/home/kamil/.local/lib/python3.9/site-packages/aiohttp/connector.py", line 1076, in _wrap_create_connection', '    raise client_error(req.connection_key, exc) from exc', "aiohttp.client_exceptions.ClientConnectorError: Cannot connect to host 192.168.1.2:61001 ssl:default [Connect call failed ('192.168.1.2', 61001)]", '2024-12-11 03:27:45,748 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-11 03:27:45,781 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-11 03:27:45,888 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-11 03:27:45,939 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', 'TP is  0.001', '2024-12-11 03:27:45,956 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.2:61001', '2024-12-11 03:27:45,978 - tno.mpc.communication.httphandlers - INFO - HTTPServer: Shutting down server task', '2024-12-11 03:27:45,980 - tno.mpc.communication.httphandlers - INFO - Server 192.168.1.1:61002 shutdown', 'Total bytes received: 1930', 'Total messages received: 5', '2024-12-11 03:27:45,986 - tno.mpc.communication.httphandlers - INFO - Client 192.168.1.2:61001 shutdown', 'Total bytes sent: 766', 'Total messages sent: 5', '2024-12-11 03:27:45,989 - tno.mpc.communication.pool - INFO - Pool shutdown.', 'Total bytes sent: 766', 'Total messages sent: 5', 'Total bytes received: 1930', 'Total messages received: 5', '\tCommand being timed: "python3.9 server_energy.py 1 870"', '\tUser time (seconds): 6.75', '\tSystem time (seconds): 0.78', '\tPercent of CPU this job got: 86%', '\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:08.66', '\tAverage shared text size (kbytes): 0', '\tAverage unshared data size (kbytes): 0', '\tAverage stack size (kbytes): 0', '\tAverage total size (kbytes): 0', '\tMaximum resident set size (kbytes): 80444', '\tAverage resident set size (kbytes): 0', '\tMajor (requiring I/O) page faults: 0', '\tMinor (reclaiming a frame) page faults: 16071', '\tVoluntary context switches: 66', '\tInvoluntary context switches: 135', '\tSwaps: 0', '\tFile system inputs: 0', '\tFile system outputs: 0', '\tSocket messages sent: 0', '\tSocket messages received: 0', '\tSignals delivered: 0', '\tPage size (bytes): 4096', '\tExit status: 0', 'PowerTOP outputing using base filename log/powertop_10mbits_2024-12-11_03-29-02.csv']
2024-12-11 03:29:32,390 - DEBUG - server all values : ['modprobe cpufreq_stats failedLoaded 0 prior measurements', 'RAPL device for cpu 0', 'RAPL device for cpu 0', 'Devfreq not enabled', 'glob returned GLOB_ABORTED', 'Preparing to take measurements', 'Taking 1 measurement(s) for a duration of 11 second(s) each.', '2024-12-11 03:28:34,575 - tno.mpc.communication.httphandlers - INFO - Serving on 192.168.1.2:61001', '2024-12-11 03:28:35,420 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-11 03:28:35,532 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-11 03:28:35,542 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-11 03:28:35,630 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', 'TP is  0.001', '2024-12-11 03:28:35,676 - tno.mpc.communication.httphandlers - INFO - Received message from 192.168.1.1:61002', '2024-12-11 03:28:35,681 - tno.mpc.communication.httphandlers - INFO - HTTPServer: Shutting down server task', '2024-12-11 03:28:35,683 - tno.mpc.communication.httphandlers - INFO - Server 192.168.1.2:61001 shutdown', 'Total bytes received: 766', 'Total messages received: 5', '2024-12-11 03:28:35,686 - tno.mpc.communication.httphandlers - INFO - Client 192.168.1.1:61002 shutdown', 'Total bytes sent: 1930', 'Total messages sent: 5', '2024-12-11 03:28:35,687 - tno.mpc.communication.pool - INFO - Pool shutdown.', 'Total bytes sent: 1930', 'Total messages sent: 5', 'Total bytes received: 766', 'Total messages received: 5', '\tCommand being timed: "python3.9 server_energy.py 0 870"', '\tUser time (seconds): 6.82', '\tSystem time (seconds): 0.70', '\tPercent of CPU this job got: 88%', '\tElapsed (wall clock) time (h:mm:ss or m:ss): 0:08.47', '\tAverage shared text size (kbytes): 0', '\tAverage unshared data size (kbytes): 0', '\tAverage stack size (kbytes): 0', '\tAverage total size (kbytes): 0', '\tMaximum resident set size (kbytes): 80540', '\tAverage resident set size (kbytes): 0', '\tMajor (requiring I/O) page faults: 0', '\tMinor (reclaiming a frame) page faults: 15958', '\tVoluntary context switches: 59', '\tInvoluntary context switches: 128', '\tSwaps: 0', '\tFile system inputs: 0', '\tFile system outputs: 0', '\tSocket messages sent: 0', '\tSocket messages received: 0', '\tSignals delivered: 0', '\tPage size (bytes): 4096', '\tExit status: 0', 'PowerTOP outputing using base filename log/powertop_10mbits_2024-12-11_03-29-02.csv']
2024-12-11 03:29:33,783 - DEBUG - All values from powertop for client: 1       424 mW  
9      5.17 mW  
39      675 uW  
Name: 7, dtype: object
2024-12-11 03:29:33,783 - DEBUG - Energy client: 3.20234525
2024-12-11 03:29:35,109 - DEBUG - All values from powertop for server: 1       545 mW  
29     1.28 mW  
Name: 7, dtype: object
2024-12-11 03:29:35,109 - DEBUG - Energy server: 4.0916372
